Composer/packages/server/src/controllers/status.ts

```diff
@@ -9,7 +9,11 @@ const getStatus = async (req, res) => {
     if (jobId) {
       const result = BackgroundProcessManager.getProcessStatus(jobId);
       if (result) {
-        res.status(result.httpStatusCode).json(result);
+        res.status(result.httpStatusCode).json({
```

Comment:
are we ok with sending the additional properties httpStatusCode and latestMessage in addition to the statusCode and message. Feels like the response will have duplicate fields for status code and message.

Reply:
I think that's ok -- the client is looking for statusCode and message only in order to display them in the UI. No harm in leaving the other fields in there as there other fields in that object as well that are used at times in the UI such as the history and log output of the background process.